pkg/networkmanager/index.html

```diff
@@ -391,6 +392,986 @@ <h2 class="panel-title" translate="yes">Unmanaged Interfaces</h2>
+            <div class="pf-c-alert__icon">
+                <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+            </div>
+            <h4 class="pf-c-alert__title"></h4>
+          </div>
+          <button class="pf-c-button pf-m-secondary" translate="yes" id="network-mac-settings-cancel">Cancel</button>
+          <button class="pf-c-button pf-m-primary" translate="yes" id="network-mac-settings-apply">Apply</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal" id="network-wifi-settings-dialog"
+       tabindex="-1" role="dialog"
+       data-backdrop="static">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title" translate="yes">WiFi</h4>
+        </div>
+        <div class="modal-body">
+          <div id="network-wifi-settings-body">
+          </div>
+        </div>
+        <div class="modal-footer">
+          <div class="alert alert-danger dialog-error" id="network-wifi-settings-error">
+            <i class="fa fa-exclamation-triangle"></i>
+            <span></span>
+          </div>
+          <button class="btn btn-default" translate="yes" id="network-wifi-settings-cancel">Cancel</button>
+          <button class="btn btn-primary" translate="yes" id="network-wifi-settings-apply">Apply</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal" id="error-popup"
+       tabindex="-1" role="dialog"
+       data-backdrop="static">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title" translate="yes">Unexpected error</h4>
+          <button class="close pficon pficon-close" data-dismiss="modal"></button>
+        </div>
+        <div class="modal-body">
+          <p id="error-popup-message"></p>
+        </div>
+        <div class="modal-footer">
+          <button class="pf-c-button pf-m-primary" data-dismiss="modal" translate>Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal" id="confirm-breaking-change-popup"
+       tabindex="-1" role="dialog"
+       data-backdrop="static">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title" translate="yes">Connection will be lost</h4>
+          <button class="close pficon pficon-close" data-dismiss="modal"></button>
+        </div>
+        <div class="modal-body">
+          <div class="pficon pficon-warning-triangle-o"></div>
+          <div id="confirm-breaking-change-text"></div>
+        </div>
+        <div class="modal-footer">
+          <button class="pf-c-button pf-m-danger"></button>
+          <button class="pf-c-button pf-m-primary" data-dismiss="modal" translate>Keep connection</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+</body><!DOCTYPE html>
```

Comment:
What's going on here?  index.html has been duplicated somehow.

Reply:
Sorry, just fixed it. Stupid mistake